### DISCUSSION TITLE
Hiding the gameplay leaderboard

### DISCUSSION THREAD

**Discussion by sineplusx** [ppy/osu#20432] [Ideas] [answered]
The new update (2022.923.0) introduced the leaderboard for solo play, which was previously only used in multiplayer matches.

In my opinion, this leaderboard design is way less attention-stealing than it currently is in stable, but there I at least have the option to hide it with the Tab key (excluding breaks). Now that the leaderboard is visible in solo gameplay, it annoys me and I currently can't get rid of it. If I want to hide it, I have to disable the whole UI which contains information I want to see, like hit error and accuracy.

~~An option in the settings for the leaderboard would be very nice to have.
Kind of like this: Leaderboard visibility (Always - Show during breaks - Never)
Making a dedicated hotkey for toggling would also be a great option.~~

Edit:
I just read some related discussions ([17978](https://github.com/ppy/osu/discussions/17978) / [20431](https://github.com/ppy/osu/discussions/20431)). If the current planning includes the leaderboard as part of the HUD elements, which can be toggled based on gameplay state, I would be perfectly happy with that.

**Comment by peppy** [comment]
Yep, more flexibility is coming soon!

**Reply by oleflock** [reply]
I am not sure if I am missing something but on that topic i found that you can turn the leaderboard off and on now but the leaderboard will only show if you have the hud on. maybe i am just odd but I like to play with no HUD and only leaderboard on. Would be cool if it would still be possible like in the stable version. If i only missed something i am sry to bother. ^^

**Comment by bdach** [comment, marked as answer]
For now, addressed by https://github.com/ppy/osu/pull/20488

**Comment by sineplusx** [comment]
Fixed by #24402 in the latest release, thank you!
